backend/kaos_backend/util/validators.py

```diff
@@ -159,3 +162,21 @@ def wrapper(*args, **kwargs):
         return function(*args, **kwargs)
 
     return wrapper
+
+
+def validate_request(function):
+    """
+    Validate the token in the header
+    """
+
+    def wrapper(*args, **kwargs):
+        token = os.getenv("TOKEN")
+        if 'Token' not in request.headers:
+            raise AuthorizationError("Token not present in the request")
+        req_token = request.headers['Token']
```

Comment:
Please use the standard [HTTP `Authorization` Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization).